.travis.yml

```diff
@@ -39,8 +39,17 @@ deploy:
     tags: true
     repo: simplabs/ember-simple-auth
 
+after_deploy:
+  - npm i -g now-pipeline
+  - now-pipeline-prune && now-pipeline --alias ember-simple-auth.now.sh
+  - cd server && now-pipeline-prune && now-pipeline --alias ember-simple-auth-server.now.sh
```

Comment:
I think we need

```yaml
on:
  tags: true
  repo: simplabs/ember-simple-auth
```

here so we only ever deploy the latest tag (which is when we also release a new version to npm).